h/buildext.py

```diff
@@ -219,80 +236,78 @@ def build_chrome(args):
     subprocess.call(subprocess_args)
 
     # Render the sidebar html
-    base_url = env['request'].route_url('index')
-    api_url = env['request'].route_url('api')
-    websocket_url = client.websocketize(env['request'].route_url('ws'))
-    sentry_dsn = env['request'].sentry.get_public_dsn('https')
-    settings = settings_dict(base_url=base_url,
-                             api_url=api_url,
-                             sentry_dsn=sentry_dsn)
+    api_url = '{}/api'.format(base_url)
+    websocket_url = websocketize('{}/ws'.format(base_url))
+    sentry_dsn = None
+
+    if args.sentry_dsn:
+        sentry_dsn = raven.Client(args.sentry_dsn).get_public_dsn()
```

Comment:
As previously discussed, this won't work if the passed DSN is a public DSN, which it will have to be on Travis:

```
$ python -c 'import raven; raven.Client("https://e06fdcb78fcd46e48ba87129ae142180@app.getsentry.com/37293").get_public_dsn()'

Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/borior/.virtualenvs/h/lib/python2.7/site-packages/raven/base.py", line 140, in __init__
    self.set_dsn(dsn, transport)
  File "/Users/borior/.virtualenvs/h/lib/python2.7/site-packages/raven/base.py", line 194, in set_dsn
    transport_registry=self._registry,
  File "/Users/borior/.virtualenvs/h/lib/python2.7/site-packages/raven/conf/remote.py", line 91, in from_string
    raise InvalidDsn('Invalid Sentry DSN: %r' % url.geturl())
raven.exceptions.InvalidDsn: Invalid Sentry DSN: 'https://e06fdcb78fcd46e48ba87129ae142180@app.getsentry.com/37293'
```


I'd suggest we explicitly name the argument `--sentry-public-dsn` and then use it directly, without validation, if passed.